src/main/java/ch/njol/skript/lang/Section.java

```diff
@@ -175,10 +181,17 @@ protected void loadOptionalCode(SectionNode sectionNode) {
 
 	@Nullable
 	public static Section parse(String expr, @Nullable String defaultError, SectionNode sectionNode, List<TriggerItem> triggerItems) {
```

Comment:
What is the goal of the changes here? It doesn't actually seem to do anything, as the resetting of annotations occurs before the SkriptParser is invoked. That is, nothing happens between saving them and resetting them.